privacy-policy.md

```
@@ -5,27 +5,34 @@
+
 Since no user data is collected, stored, or transmitted, there are no security risks associated with data exposure.
 
 ### 6. Changes to This Policy
+
 Any updates will be posted on this page. (don't plan to make it intrusive though)
 
 ### 7. Contact
+
 If you have any questions about this privacy policy, please contact at:
-📩 abhishekaggarwal1995@gmail.com 
+📩 abhishekaggarwal1995@gmail.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)